### PR TITLE
Fixed a few callouts (added some, reverted some)

### PR DESCRIPTION
**Pull request by sirlantis**

### CODE REVIEW

File: docs/platform/software-version-hardware-version.md [hunk 1]
```diff
@@ -19,10 +19,8 @@ On Android, Memfault uses the `ro.product.board` system property as the device's
 hardware version by default. From the
 [Android product documentation](https://source.android.com/setup/develop/new-device#build-layers):
 
-:::note
-The board/device layer represents \[...\] the bare schematics of a product.
-These include the peripherals on the board and their configuration.
-:::
+> The board/device layer represents \[...\] the bare schematics of a product.
+> These include the peripherals on the board and their configuration.
```
Comment (martijnthe) [inline docs/platform/software-version-hardware-version.md:23]: Why revert this? It looked fine to me?

**Review by martijnthe — APPROVED** [review]

File: docs/platform/software-version-hardware-version.md (hunk 1 above)
Comment (sirlantis) [reply docs/platform/software-version-hardware-version.md:23]: I noticed that this one is actually a quote, introduced with a "From the Android product documentation:" above.

Comment (martijnthe) [reply docs/platform/software-version-hardware-version.md:23]: Ah, right, thanks!

File: docs/android/android-releases-guide.md [hunk 2]
```diff
@@ -160,8 +160,7 @@ INFO: You can view in the UI here:
     <Link to Release in UI>
 ```
 
-#### Tip
-
+:::note Tip
```
Comment (sirlantis) [inline docs/android/android-releases-guide.md:163]: I used `:::note Tip` instead of `:::tip` here because a large green block is visually jarring.

**Review by rjcase — APPROVED** [review]
Thanks!

**Pull request merged**